### PR TITLE
Vinodhmohan91 dev 3

### PR DESCRIPTION
**Pull request by vinodhmohan91**
Added and edited content to show the steps to save the spam filter model in both Watson Studio Local and Remote HDP cluster. Also, has steps to do a batch scoring and model evaluation.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,11 +40,13 @@ As mentioned earlier, this code pattern offers three examples of how to develop,
   `"Spam Filter on local spark"`
     - Load the spam collection dataset using Spark context in Watson Studio Local.
     - Use Spark Data Pipeline to extract the TF-IDF features and use Spark MLlib to train the Spam Filter pyspark model locally.
+    - Save the Spam Filter pyspark model in Watson Studio Local.
 
   `"Spam Filter on remote spark"`
     - Push the spam collection dataset to the remote HDFS user directory in the HDP cluster.
     - Connect to the remote Spark context in the HDP cluster via the Hadoop Integration service using the sparkmagics library.
     - Uses `%%spark` to run the remote Spark context to load, extract and train the Spam Filter pyspark model in the HDP cluster.
+    - Save the Spam Filter pyspark model in HDP cluter and import the model into Wastson Studio Local.
```
Comment (rhagarty) [inline README.md]: typos - cluster and Watson

File: README.md [hunk 2]
```diff
@@ -372,6 +375,68 @@ Connect to remote Spark in HDP cluster and run the LRModelScikit function using
...
 
+#### 8. Save the Spam Filter Pyspark Model in Watson Studio Local
+
+Use the save function in `dsx_ml.ml` library to save the Spam Filter pyspark model with the name `LRModel_SparkLocal`.
+
+![](doc/source/images/save-model-local.png)
+
+#### 9. Save the Spam Filter Pyspark Model in remote HDP cluster
+
+After training the model (step 3) in remote HDP cluster, the model exists in the memory of the remote session. Write the model to a HDFS location and copy it to local node as show below.
```
Comment (rhagarty) [inline README.md]: "as shown below"

File: README.md [hunk 3]
```diff
@@ -372,6 +375,68 @@ Connect to remote Spark in HDP cluster and run the LRModelScikit function using
...
+
+![](doc/source/images/save-model-remote-4.png)
+
+You can now see the saved model `LRModel_SparkRemote` in the `Models` tab. 
+
+![](doc/source/images/models-screenshot.png)
+
+#### 11. Batch score the Spam Filter Pyspark Model
+
+Click any one of the above models to test and score. Provide a test message as `Input` and press `submit` to do a real time scoring.
```
Comment (rhagarty) [inline README.md]: "real-time"

File: README.md [hunk 4]
```diff
@@ -372,6 +375,68 @@ Connect to remote Spark in HDP cluster and run the LRModelScikit function using
...
+
+![](doc/source/images/real-time-scoring.png)
+
+Select the `Execution Type`, `Input data set` and `Output data set` and click `Generate batch script` to do a batch scoring.
```
Comment (rhagarty) [inline README.md]: "do a batch scoring run" or maybe "run a batch scoring job"?

**Review by rhagarty — CHANGES_REQUESTED** [review]
couple small nits

File: README.md (hunk 1 above)
Comment (vinodhmohan91) [reply README.md]: Done

File: README.md (hunk 2 above)
Comment (vinodhmohan91) [reply README.md]: Done

File: README.md (hunk 3 above)
Comment (vinodhmohan91) [reply README.md]: Done

File: README.md (hunk 4 above)
Comment (vinodhmohan91) [reply README.md]: Done

**Comment by vinodhmohan91** [comment]
Made the changes requested. Please review and merge.

**Review by rhagarty — APPROVED** [review]

**Pull request merged**